### DISCUSSION TITLE
Can't access Google Gemini

### DISCUSSION THREAD

**Discussion by thesmol** [angristan/wireguard-install#494] [Help] [answered]
I am from Russia so almost services are not available to me. I have a German VPS on Ubuntu 20.04 from HOSTKEY, with the standard default installation, everything works, but I can't access Google Gemini via VPN - it says unavailable. At the same time, the same Copilot from Microsoft works stably.
Everything I tried to change in the default installation - change DNS from 1.1.1.1.1, 1.0.0.1 to 8.8.8.8.8, 8.8.4.4 - didn't work.

I'm not a very experienced user, so I don't quite understand why I have this problem and how to fix it.

**Comment by Sirherobrine23** [comment]
probably google blocking HOSTKEY, and many other cloud platforms to access Gemini, and another google products, check this

**Reply by thesmol** [reply]
Hello, thank you for your reply. 
Check what?

**Reply by Sirherobrine23** [reply, marked as answer]
Sorry, Gemini is blocked for several cloud platforms, checked in my Oracle cloud and aws vps

**Reply by CXwudi** [reply]
My hetzner VPS is also blocked recently 😕. I guess time to use OpenRouter for now...
